gui/accelmap.py

```diff
@@ -279,10 +296,44 @@ def _get_accel_map_entries(cls):
     ## Search
 
     def _view_search_equal_cb(self, model, col, key, it):
-        # case-insensitive check if the key is a substring of the search col
-        is_sub = _udecode(key).lower() in _udecode(model.get_value(it, col))
+        is_sub = self._filter_check(model, it, key)
         return not is_sub  # inverted sense (as in equality w. strcmp)
 
+    # Filter
+
+    def _filter_check(self, model, iter, search_key):
+        if search_key:
+            txt = search_key
+        else:
+            txt = self._filter_txt
+
+        if not txt:
+            return True
+
+        reg_sep = r'(?:\+(?!\w(?:$|\s))|\s)'
```

Comment:
Add a short comment explaining the logic here. It usually helps when you come back to the code half a year later, especially if that code involves python regexes.